template/README.md

```diff
@@ -1,16 +1,16 @@
 # %TITLE%
 
 ## Development instructions
-1. In the root directory, run `npm install` to install all the project dependencies
+
+1. In the root directory, run `npm ci` to install all the project dependencies
```

Comment:
It's worth noting here, that we now include a package-lock.json.

While this means that patch version updates won't automatically be consumed, I think this will do a lot of good if end-users ever face issues out-of-the-box. 

The consumption model for this project is almost always via `npx`, so if there ever is a critical change, it should be easy for us to make a change in tram-one-express and release that out to people.